python/deadlinesubmissionsettings/app.py

```diff
@@ -191,9 +191,13 @@ def create_machine_limit_options(self):
             self.machine_list.addItem(name)
 
     def create_pools_options(self):
-        pools = lib.get_pool_list()
-        for pool in pools:
+        pools = ["-"]
+        pools.extend(lib.get_pool_list())
+
+        for pool in pools[1:]:
             self.primary_pool.addItem(pool)
+
```

Comment:
How about not extending the pools above but just doing  `self.secondary_pool.addItem("")` or alike?

That way it also makes sense to comment above that line and say something like:
```python
# Deadline's secondary pool has an extra empty value which
# is the unassigned second pool. The "none" entry instead
# for the secondary pool actually assigns all machines in the
# "none" pool to be valid for rendering, even if primary
# pool is assigned.
```

Maybe too verbose, but could be useful for future devs?

Reply:
The pool system is an order of priority per slave. 

Slave A is assigned to pools `redshift, houdini, none`; it will prioritise jobs with primary pool set to `redshift` over jobs with primary pool set the `houdini`. If there are no jobs left with primary pool `redshift` it will run those with `houdini` till another it is done or another job with primary  pool `redshift` 

Hope this explains it a bit better

Reply:
Ah, if it's just an explanation of how Deadline works, then it's best to just state they need to check Deadline docs - what we're solving here is mostly about the difference in having "none" and it behaving oddly. So the priority thing is less important.